### ISSUE TITLE
GeoJsonReader cannot read in an array property

### ISSUE THREAD

**Issue by jnicho02** [NetTopologySuite/NetTopologySuite#61]
When you read in some geojson that has a property that is an array the GeoJsonReader throws System.ArgumentException : Expected token '}' not found.

The GeoJsonWriter *is* able to write out geojson with an array as a property.

```C#
        [Test]
        public void GeoJsonDeserializeFeatureCollectionArrayPropertyTest()
        {
            const string json = @"
{
  ""type"": ""FeatureCollection"",
  ""features"": [
    {
      ""type"": ""Feature"",
      ""geometry"": {
        ""type"": ""Point"",
        ""coordinates"": [
          1.0,
          2.0
        ]
      },
      ""properties"": {
        ""foo"": [
            {
              ""zee"": ""xyz""
            }
          ]
      }
    }
  ]
}
";
            GeoJsonReader reader = new GeoJsonReader();
            FeatureCollection coll = reader.Read<FeatureCollection>(json);
            Assert.IsNotNull(coll);

            GeoJsonWriter writer = new GeoJsonWriter();
            string s = writer.Write(coll);
            Assert.IsNotNull(s);
        }
```

**Comment by jnicho02** [comment]
Getting there. I see you are testing an array-array as well.

**Comment by FObermaier** [comment]
Fixed with 06b19a34034d96b13c71488c3a56bd9daa223905

**Comment by DGuidi** [comment]
+1
